C++/Navio/Common/gpio.cpp

```diff
@@ -32,12 +31,14 @@
 #define GPIO_GET(g)         (*(_gpio+13)&(1<<g)) // 0 if LOW, (1<<g) if HIGH
 
 #define MAX_SIZE_LINE       50
+#define RPI_SIZE            12
```

Comment:
Bad name. I would not understand what this means without knowing the context.

Reply:
Is now better?